spec/fixtures/application/1.0/application_completed.json

```diff
@@ -0,0 +1,48 @@
+{
+  "id": "10fd999b-0327-4826-a5f4-ac348c70ede4",
+  "schema_version": 1.0,
+  "reference": 6000003,
+  "created_at": "2023-01-11T13:25:00.000+00:00",
+  "submitted_at": "2023-01-11T14:10:00.000+00:00",
+  "date_stamp": "2023-01-11T13:28:00.000+00:00",
+  "status": "submitted",
+  "review_status": "assessment_complete",
```

Comment:
Is this `review_status` supposed to be here?

Reply:
I'm still a touch lost on this one, but for now yes, as I am checking against it.

I want to come back and do a refactor after I've got this working, and work out how all statuses are going to be stored. I think we might need a meeting to rethink